src/steps/03-Background/CurrentContract/CurrentContract.spec.ts

```diff
@@ -27,29 +20,19 @@ describe("Testing CurrentContract Component", () => {
     savedData:{ current_contract_exists: "YES" }
   }
 
-  const mockRoute = {
-    name: "Current_Contract_Details",
-    params: {
-      id: 1
-    }
-  }
-  const mockRouter = {
-    push: jest.fn(),
-  }
-  
 
-  beforeEach(() => {
-    vuetify = new Vuetify();
-    wrapper = mount(CurrentContract, {
-      localVue,
-      vuetify,
+  const wrapper: VueWrapper = shallowMount(CurrentContract, {
```

Comment:
Test is making api calls based on the `AcquisitionPackage`, please mock these to prevent the calls from being made